dotnet/src/SemanticKernel.Skills/Skills.Memory.Qdrant/QdrantVectorDbClient.cs

```diff
@@ -0,0 +1,374 @@
+
+        if (data.Vectors.Count == 0)
+        {
+            this._log.LogWarning("Vector not found: {0}", pointId);
+            return null;
+        } */
+
+        /*var record = new QdrantVectorRecord<TEmbedding>(
+            new Embedding<TEmbedding>(data.Vectors.First().Vector.ToArray()),
+            data.Vectors.First().ExternalPayload,
+            data.Vectors.First().ExternalTags);
+        this._log.LogDebug("Vector found}");
+
+        return new DataEntry<QdrantVectorRecord<TEmbedding>>(Base64Decode(pointId), record);
+        */
+        return null;
+    }
+
+    public async Task DeleteVectorAsync(string collectionName, string key)
+    {
+        var pointId = Base64Encode(key);
+        this._log.LogDebug("Deleting vector by point ID {0}", pointId);
+
+        Verify.NotNullOrEmpty(collectionName, "Collection name is empty");
+        Verify.NotNullOrEmpty(pointId, "Qdrant point ID is empty");
+
+        /*using var request = DeleteVectorsRequest
+            .DeleteFrom(collectionName)
+            .DeleteVector(pointId)
+            .Build();
+        await this.ExecuteHttpRequestAsync(request);*/
+    }
+
+    public async Task UpsertVectorAsync(string collectionName, DataEntry<QdrantVectorRecord<TEmbedding>> vectorData)
+    {
+        this._log.LogDebug("Upserting vector");
+        Verify.NotNull(vectorData, "The vector is NULL");
+
+        /*DataEntry<QdrantVectorRecord<TEmbedding>>? existingRecord = await this.GetVectorByIdAsync(collectionName, vectorData.Key);
+
+        // Generate a new ID for the new vector
+        if (existingRecord == null)
+        {
+            return;
+        }
+
+        using var request = CreateVectorsRequest<TEmbedding>
+            .CreateIn(collectionName)
+            .UpsertVector(Base64Encode(vectorData.Key), vectorData.Value!).Build();
+        var (response, responseContent) = await this.ExecuteHttpRequestAsync(request);
+
+        if (response.StatusCode == HttpStatusCode.UnprocessableEntity
+            && responseContent.Contains("data did not match any variant of untagged enum ExtendedPointId", StringComparison.OrdinalIgnoreCase))
+        {
+            throw new VectorDbException("The vector ID must be a GUID string");
+        }
+
+        try
+        {
+            response.EnsureSuccessStatusCode();
+        }
+        catch (Exception e)
+        {
+            this._log.LogError(e, "Vector upsert failed: {0}, {1}", e.Message, responseContent);
+            throw;
+        } */
+    }
+
+    public async IAsyncEnumerable<(QdrantVectorRecord<TEmbedding>, double)> FindNearesetInCollectionAsync(
+        string collectionName,
+        Embedding<TEmbedding> target,
+        int top = 1,
+        string[]? requiredTags = null)
+    {
+        this._log.LogDebug("Searching top {0} closest vectors in {1}", top);
+
+        Verify.NotNull(target, "The given vector is NULL");
+
+        /*using HttpRequestMessage request = SearchVectorsRequest<TEmbedding>
```

Comment:
please remove unused, commented code

Reply:
@awharrison-28 I think this is from bad merge. Let me do a clean pull now and update files.